docs/devel/networking.md

```diff
@@ -378,4 +380,4 @@ which implicitly highly encourages IPv6 communication towards the VM instance.
 
 On a final note, there is a difference that impacts the user experience when
 using masquerade binding for IPv6 addresses; the VMI IP must be [manually
-configured by the user](https://kubevirt.io/user-guide/#/creation/interfaces-and-networks?id=masquerade-ipv6-support).
\ No newline at end of file
+configured by the user](https://kubevirt.io/user-guide/#/creation/interfaces-and-networks?id=masquerade-ipv6-support).
```

Comment:
I don't understand what change was done here and why this line is marked as changed. Anyway, the link is broken and should be updated to https://kubevirt.io/user-guide/#/creation/interfaces-and-networks?id=masquerade-ipv4-and-ipv6-dual-stack-support

Reply:
The editor added a newline by the end of the doc. With your permission, I'd fix the broken URL in a follow-up PR